subworkflows/download_databases.nf

```diff
@@ -126,6 +129,25 @@ workflow DOWNLOAD_DATABASES {
             )
         }
 
+        if (pseudofinder_file.exists()) {
+            log.info("Pseudofinder database exists, or at least the expected folder.")
+            fileName = pseudofinder_file.getName()
+            match = (fileName =~ /uniprot_sprot_(\d{4}_\d{2})\.fasta/)
+            if (match) {
+                pseudofinder_db_version = match[0][1]
+            } else {
+                log.warn("Failed to extract version from file name ${fileName}")
+                pseudofinder_db_version = "Unknown"
+            }
```

Comment:
I the other dbs we have a VERSION file with the corresponding version, let's keep it like that pleasw